sources/api/bootstrap-containers/src/main.rs

```diff
@@ -133,16 +136,19 @@ fn usage() {
         mark-bootstrap
 
     Global arguments:
+        [ --config-path PATH ]
         [ --socket-path PATH ]
```

Comment:
Is the --socket-path still required? If not we should either 

a) note that it is deprecated and give a warning if the command line arg is used

or, preferably

b) remove the argument from the cli

Edit: Oh I see it still used. Probably to mount it into the container it is starting.